src/js/utils.js

```diff
@@ -47,15 +47,50 @@ const changeAnchorLink = (anchorElement, { user, repository, typeOfSearch, date
   }
 }
 
-export const validateForm = (formData, anchorElement) => {
-  const filterData = createSearchFilter(formData);
+export const validateForm = () => {
+  const fielValid = {
+    type_of_search: {
+      required: true,
+    },
+    input_user: {
+      required: true,
+    },
+    input_repository: {
+      required: true,
+    },
+    input_condition: {
+      required: true,
+    },
+    input_date: {
+      required: true,
+    },
+  };
+  const fiels = document.querySelectorAll(
+    ".box.d-inline input,.box.d-inline select"
+  );
 
-  const isValid = filterData.user && filterData.repository && filterData.typeOfSearch && filterData.date;
+  let bool = true;
+  for (var i in fiels) {
```

Comment:
Please considerate to change the name "i" for another name, "i" for standard is the index of one array or similar, but this is not one index of array else one item of fiels, what do you think about to use field

```suggestion
  for (var field in fields) {
```